examples/TransactionGenerator.hs

```diff
@@ -1,13 +1,15 @@
 {-# LANGUAGE BangPatterns                    #-}
 {-# LANGUAGE DataKinds                       #-}
-{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveAnyClass                  #-}
```

Comment:
Unfortunately something about this file doesn't allow `stylish-haskell` to succeed on it (otherwise these `#-}` would have been reformatted back).